src/profile-logic/import/instruments/MaybeCompressedReader.js

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+import * as pako from 'pako';
+
+class MaybeCompressedDataReader {
```

Comment:
same thing as in other files: please add comments, and please export it directly as a named export, not as a default